src/nupic/algorithms/Connections.cpp

```diff
@@ -557,42 +557,6 @@ void Connections::bumpSegment(const Segment segment, const Permanence delta) {
 }
 
 
-void Connections::destroyMinPermanenceSynapses(
-                              const Segment segment, Int nDestroy,
```

Comment:
this is some mistake, was added by #446

Reply:
..all the connections changes seem out of place. try `git checkout -- master src/nupic/..../Connections.* `